content/en/docs/concepts/overview/working-with-objects/kubernetes-objects.md

```diff
@@ -26,8 +26,7 @@ To work with Kubernetes objects--whether to create, modify, or delete them--you'
 
 ### Object Spec and Status
 
-Every Kubernetes object includes two nested object fields that govern the object's configuration: the object *spec* and the object *status*. The *spec*, which you must provide, describes your desired state for the object--the characteristics that you want the object to have. The *status* describes the *actual state* of the object, and is supplied and updated by the Kubernetes system. At any given time, the Kubernetes Control Plane actively manages an object's actual state to match the desired state you supplied.
-
+Almost every Kubernetes object includes two nested object fields that govern the object's configuration: the object *spec* and the object *status*. The *spec*, which you must provide, describes your desired state for the object--the characteristics that you want the object to have. The *status* describes the *actual state* of the object, and is supplied and updated by the Kubernetes system. At any given time, the Kubernetes {{< glossary_tooltip text="control plane" term_id="control-plane" >}} actively manages an object's actual state to match the desired state you supplied.
```

Comment:
tweak "The *spec*, which you must provide," ?

Reply:
What kind of tweak did you have in mind @tengqm?

Reply:
Maybe something like:

The *spec* is a mandatory field. It describes your desired state for the object.

Reply:
Thing is: `spec` isn't mandatory (counter-examples: Namespace, ConfigMap, Secret) even though it is common. So I'm wary of misleading new readers and that was why I wanted to revise the wording here.

:thought_balloon: 
Maybe:
```markdown
Almost every Kubernetes object includes two nested object fields that govern
the object's configuration: the object *`spec`* and the object *`status`*.
For objects that have a `spec`, you have to set this when you create the object,
providing a description of the characteristics you want the resource to have:
its _desired state_.

The `status` describes the _current state_ of the object, supplied and updated
by the Kubernetes and its components. The Kubernetes
{{< glossary_tooltip text="control plane" term_id="control-plane" >}} continually
and actively manages every object's actual state to match the desired state you
supplied.
```

How about that?

Reply:
@tengqm how does that suggested wording look to you?

Reply:
Looks pretty good to me, I'd say. :)